libbeat/docs/outputconfig.asciidoc

```diff
@@ -521,6 +544,18 @@ The maximum number of files to save under <<path>>. When this number of files is
 oldest file is deleted, and the rest of the files are shifted from last to first. The default
 is 7 files.
 
+===== bulk_max_size
+
+The maximum number of events to bulk into batches. If the Beat sends single events, the events
+are collected into batches. If the Beat publishes a large batch of events (larger than the value
+specified by `bulk_max_size`), the batch is split.
+
+Specifying a larger batch size can improve performance by lowering the overhead of sending events. 
+However big batch sizes can also increase processing times.
+
+The default is 200.
```

Comment:
The diff of the changes is misleading here. This section is actually under File output, not Redis output. I didn't add the option to the Redis section because it wasn't in the list of outputs that you mentioned previously. You listed Elasticsearch, logstash, console, and file. Do I also need to add the description of bulk_max_size to Redis?

Also, I just want to confirm that I should change the default value shown for all outputs but elasticsearch, which (I think) still sets the default to 50. Is that correct?